liminal/build/service/python_server/liminal_python_server.py

```diff
@@ -27,7 +27,10 @@ def start_server(yml_path):
         __start_server(yaml.safe_load(stream))
 
 
+# noinspection PyUnresolvedReferences
 def __start_server(config):
+    globals()['request'] = __import__('flask').request
```

Comment:
why are you doing the import in this way?

Reply:
I need the lambda below to pass request data to user code, and this was the only way I could load it and have it available when the lambda below runs.